cppcache/CMakeLists.txt

```diff
@@ -59,6 +59,8 @@ elseif ("Windows" STREQUAL ${CMAKE_SYSTEM_NAME} )
   set( _WIN32 1 )
 endif()
 
+get_target_property(JAVAOBJECT_JAR_PATH javaobject JAR_FILE)
```

Comment:
This project doesn't depend on javaobject so it is relying on ordering of targets to complete correctly. Make this target depend on the Javadoc target. Also, does the root level project for C++ really need this? I imagine it is really just for integration tests. It should really be moved there.